frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -2548,9 +2548,6 @@ def identity(self, resulting_type: ConversionType, x: Conversion) -> Conversion:
             and x.is_signed == resulting_type.is_signed
         )
 
-        if resulting_type.bit_width == x.bit_width:
-            return x
```

Comment:
Didn't we said we'd keep this?

Reply:
yes the real work has already be merged. that commit can be removed from this draft

Reply:
Removed from PR